homeassistant/components/fan/services.yaml

```diff
@@ -211,3 +211,34 @@ wemo_reset_filter_life:
     entity_id:
       description: Names of the WeMo humidifier entities (1 or more entity_ids are required).
       example: 'fan.wemo_humidifier'
+
+vallox_set_profile_fan_speed_home:
```

Comment:
Integration specific services should be added to the integration package and registered under the integration domain.

Reply:
Will change.
Just for my own understanding: I tried to start from an existing integration, in this case xiaomi miio and wemo, where they did it that way. Was I wrong to assume this is the same case, or did these rules not apply back when they integrated it?

Reply:
Those integrations are not the best examples, since they don't always follow our current standards and architecture.

Our architecture decisions have evolved and there are still some old integrations that don't follow them.